standard/table.lua

```diff
@@ -80,8 +80,45 @@ function Table.copy(tbl)
 	return result
 end
 
-function Table.deepCopy(tbl)
-	return mw.clone(tbl)
+--[[
+Recursively copies a table.
+
+options.copyMetatable
+If enabled, deep copies the metatable of tables. Disabled by default.
```

Comment:
```suggestion
If enabled, deep copies the metatable of tables. Disabled by default. Doing so omits the metatables. The contents provided by any metatables are still copied.
```

Reply:
ah you meant the comment in code. I added it now